sphinx/templates/latex/content.tex_t

```diff
@@ -1,12 +1,23 @@
 %% Generated by Sphinx.
 \def\sphinxdocclass{<%= docclass %>}
+<% if latex_engine == 'lualatex' -%>
+\IfFileExists{luatex85.sty}
+ {\RequirePackage{luatex85}}
+ {\ifdefined\luatexversion\ifnum\luatexversion>84\relax
+  \PackageError{sphinx}
+  {** With this LuaTeX (\the\luatexversion),Sphinx requires luatex85.sty **}
+  {** Add the LaTeX package luatex85 to your TeX installation, and try again **}
+  \endinput\fi\fi}
+<% endif -%>
 \documentclass[<%= papersize %>,<%= pointsize %><%= classoptions %>]{<%= wrapperclass %>}
 \ifdefined\pdfpxdimen
    \let\sphinxpxdimen\pdfpxdimen\else\newdimen\sphinxpxdimen
 \fi \sphinxpxdimen=<%= pxunit %>\relax
 <%= passoptionstopackages %>
 <%= geometry %>
+<% if latex_engine == 'pdflatex' -%>
```

Comment:
I will update documentation, when decision about inputenc key is taken:

- I found Sphinx's `html_output_encoding` but no equivalent for latex output, I don't know if it is possible to produce latex file for example in `iso-latin-1` encoding.

- For English with only 7bit-ascii source, dropping usage of `inputenc` makes sense, as it frees the latex log of dozens of lines written by un-needed `\usepackage[utf8]{inputenc}`

- I thus made error by removing `\ifdefined\DeclareUnicodeCharacter...`  from `'utf8extra'`, because if user sets `inputenc` key (which has already been documented) to empty then indeed `\DeclareUnicodeCharacter` is not defined.

If there is no way to customize `.tex` document encoding which is always UTF-8, then better situation would be with `inputenc` key left undocumented: currently `utf8extra` is undocumented. Advanced user would set both to empty strings simultaneously if they want. But I don't know what is best.